meta/meta_test.go

```diff
@@ -47,11 +47,17 @@ func (s *testSuite) TestMeta(c *C) {
 	defer testleak.AfterTest(c)()
 	store, err := mockstore.NewMockStore()
 	c.Assert(err, IsNil)
-	defer store.Close()
+	defer func() {
+		// to fix: close fails
+		_ = store.Close()
```

Comment:
Do we need to fix it?

Reply:
I am not very familiar with the code in meta package, but I assume `store.Close()` should be err nil, however it returned err in this line. So either we ignore the err or somewhat fix the test?

Reply:
Look at this code, I  think we can remove line57-60. Because we commit this `txn` on [line](https://github.com/pingcap/tidb/pull/22995/files#diff-5f21eb0f7d7a4cead4c91ed14a9041bd767af9d32657d5fbd66356034284916eR272).

Reply:
Removed and test passed.